content/applications/inventory_and_mrp/manufacturing/subcontracting/dropship_subcontracting_lead_times.rst

```diff
@@ -0,0 +1,187 @@
+==================================
+Dropship subcontracting lead times
+==================================
+
+.. |PO| replace:: :abbr:`PO (Purchase Order)`
+.. |RfQ| replace:: :abbr:`RfQ (Request for Quotation)`
+.. |BoM| replace:: :abbr:`BoM (Bill of Materials)`
+
+In Odoo, lead times are used to predict how long it takes to complete a certain action. For example,
+a *delivery lead time* can be set for a purchased product, which specifies the number of days it
+usually takes for the product's vendor to delivery the product to the purchasing company.
+
+For subcontracted products specifically, delivery lead times can be configured to take into account
+the amount of time required for the subcontractor to manufacture a product. Doing so allows the
+contracting company to better predict the delivery dates of subcontracted products.
+
+Certain subcontracted products require the contracting company to supply the subcontractor with
+manufacturing components. In this case, a *manufacturing lead time* can be used in addition to the
+delivery lead time, to generate the date on which the subcontractor must receive the required
+components, in order to manufacture the product and deliver it on time.
+
+In cases where components are dropshipped to the subcontractor, an additional *delivery lead time*
+can be set for each component, to generate the date that the component must be purchased, so that it
+can be dropshipped to the subcontractor by the date on which manufacturing must begin.
```

Comment:
A lot of this makes sense to me, and the explanation is well-written! I am however, still a little lost about the distinction between the product delivery lead time and the component delivery lead time.

Checking my understanding here:
- The product delivery lead time is the duration between confirmation of the sales order of the product to delivery to the customer, right? 
- Component lead time is the duration between placing the PO and its arrival at the subcontractor location, for manufacturing?

It might be helpful for readers to understand the various lead time definitions and how it impacts the subcontracting workflow to have textbook-like definitions of the lead times in one place, and how the lead times are used to schedule various supply chain operations so customers get them in time. This is just a suggestion, coming from someone who's a little confused :sweat_smile: